cluster_cleanup.sh

```diff
@@ -7,6 +7,10 @@ source lib/common.sh
 # Delete cluster
 if [ "${EPHEMERAL_CLUSTER}" == "kind" ]; then
   sudo su -l -c "kind delete cluster  || true" "${USER}"
+  # Kill and remove the running ironic containers
+  if [ -d "$BMOPATH" ]; then
```

Comment:
I would rather check for the executable itself. you might have an old fork without the executable

Reply:
How about now?